_posts/2020-06-10-experiences-using-liberating-structures.markdown

```diff
@@ -0,0 +1,103 @@
+* Microstructures are the smaller parts such as; meeting rooms, brainstorm and discussions. This is where and how discussions take place.
+* Structural elements facilitate the microstructures. For example, whiteboards, time allocations and agendas.
+
+
+All the structures play an important role, as each builds upon the smaller elements. Small changes on the lower levels over time can significantly effect the bigger structures. I have noticed for example, that virtual meetings seem to have a smaller pool of ideas and converge to a single idea much faster compared to in person meetings. Possibly, due to the fact virtual meetings only have one person speaking and it can be difficult to edge in. This reduces the pool of ideas and the best idea may not be chosen.
+
+It's important to note that each level of the structures listed above include both tangible and intangible elements. Liberating structures are specifically different ways to have a meeting/discussion/report. This puts them in the category of intangible microstructures.
+
+## Conventional Structures
+
+There are five main conventional structures that people use:
+* Presentations
+* Status Reports
+* Managed Discussions
+* Open Discussions
+* Brainstorms
+
+These are just the default options for information sharing, however they have a few problems as the first three are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. The final two (open discussions and brainstorms) are often too loose and disorganised to drive actual outcomes. 
+
+## Liberating Structures
+
+Liberating structures are 33 alternatives to the conventional structures noted above. They aim to structure your meeting to get everyone involved whilst not inhibiting anyone.
+
+The 33 have different use cases and often have different outcomes, so you picking the right tool for the job is required here. Many of them leverage the '1-2-4-All' structure, where you think alone, in pairs, in quads and then share back with the rest of the group.
+
+![](/assets/images/liberating-structures.png)
+*List of the structures taken from - [Liberating Structures][liberatingstructures]*
+
+## The Experiences
+
+This post will be updated as I use more of the structures, and if my views of the structures significantly change as I have more experince with them.
+
+I have split them into two groups, one when I facilitated the meeting and another when I was a participant.
+
+#### Contents
+
+##### [Facilitator](#f)
+1. [Wicked Questions](#f-wicked-questions)
+
+<!---
+##### [Participant](#p)
+1. [Wicked Questions](#p-wicked-questions)
+-->
+
+
+#### <a name="f"/>Acted as Facilitator
+
+<ul>
+  <details>
+  <summary><a name="f-wicked-questions"/><li><strong>Wicked Questions</strong> - Used to find the opposing strategies that both need to be pursued to be successful.</li></summary>
```

Comment:
Perhaps add in here how to use the method too i.e. 

"Used to find the opposing strategies that both need to be pursued to be successful, by..."